### PR TITLE
fixed the steam social button

### PR DESCRIPTION
**Pull request by KyleBreedlove**
Fixed the Steam Social button that was not showing up.

Issue: 1000 "Login with Steam Social button is not visible"

### CODE REVIEW

**Review by glebkema — CHANGES_REQUESTED** [review]
Thanks for your contribution! Your code solves the problem, but your need to edit the `.less` files and compile them with the `npm run compile` command instead of editing the `.css` files directly.

Look for the detailed instructions here:
https://github.com/sButtons/sbuttons/blob/master/CONTRIBUTING.md#instructions

**Review by glebkema — CHANGES_REQUESTED** [review]
< please remove this duplicate comment >

**Comment by KyleBreedlove** [comment]
My bad, completely glossed over that part. Should I create a new request and do it the correct way?

**Comment by glebkema** [comment]
The best way is to add a new commit to this pull request.

**Review by shahednasser — CHANGES_REQUESTED** [review]
Can you please remove all changes in `package-lock.json` and `package.json`?

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Logo made by [Harshit Sharma](https://www.github.com/icoderharshit)
 
 [![GitHub license](https://img.shields.io/github/license/Naereen/StrapDown.js.svg)](./LICENSE.md)
  [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](./CONTRIBUTING.md) [![Open Source Love svg1](https://badges.frapsoft.com/os/v1/open-source.svg?v=103)](./CONTRIBUTING.md) <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
-[![All Contributors](https://img.shields.io/badge/all_contributors-161-orange.svg?style=flat-square)](./CONTRIBUTORS.md)
+[![All Contributors](https://img.shields.io/badge/all_contributors-162-orange.svg?style=flat-square)](./CONTRIBUTORS.md)
```
Comment (shahednasser) [inline README.md:9]: remove this change

**Review by shahednasser — CHANGES_REQUESTED** [review]

**Comment by shahednasser** [comment]
Great work @KyleBreedlove thank you!

@all-contributors please add @KyleBreedlove for code and bug

**Pull request merged**